### ISSUE TITLE
Downloads fail on some networks with only peer protocol support (mobile)

### ISSUE THREAD

**Issue by tzarebczan** [lbryio/lbry-sdk#2238]
## The Issue
We've seen this happen a few times for mobile users on mobile networks. Not sure why they can't communicate with the reflector peer. On wifi, they can. 

```
2019-06-17 11:12:03,892 INFO     lbrynet.stream.stream_manager:388: starting download for lbry://Why-I-Quit-YouTube#dd87adb49b33f80477050c56bf937880f8f4d18c
2019-06-17 11:12:03,895 INFO     lbrynet.stream.managed_stream:274: start downloader for stream (sd hash: c001e9bfc74568d85fa4e93a36472639f4e76682302c664fe3444ba09bfa0c6c18951696b076293f4d20d61cff7349aa)
2019-06-17 11:12:04,003 INFO     lbrynet.stream.downloader:102: searching for peers for stream c001e9bfc74568d85fa4e93a36472639f4e76682302c664fe3444ba09bfa0c6c18951696b076293f4d20d61cff7349aa
2019-06-17 11:12:33,945 WARNING  lbrynet.extras.daemon.Daemon:937: Error downloading lbry://Why-I-Quit-YouTube#dd87adb49b33f80477050c56bf937880f8f4d18c: Failed to download sd blob c001e9bfc74568d85fa4e93a36472639f4e76682302c664fe3444ba09bfa0c6c18951696b076293f4d20d61cff7349aa within timeout
```

## System Configuration

 

- LBRY Daemon version:
- LBRY App version:
- LBRY Installation ID:
- Operating system:

**Comment by tzarebczan** [comment]
Even with DHT enabled for the same user, it's not able to find any peers on 4444. Possibly these ports, including the reflector one, are blocked on some mobile networks. Not sure if a different port would help yet, but I can try the user run another test with my node on different port. 

```
019-06-17 14:08:44,105 INFO     lbrynet.wallet.manager:153: Wallet at /storage/emulated/0/Android/data/io.lbry.browser/files/lbryum/wallets/default_wallet is empty, generating a default account.
2019-06-17 14:08:44,496 INFO     lbrynet.extras.daemon.Components:345: start the dht
2019-06-17 14:08:44,498 INFO     lbrynet.extras.daemon.Components:370: Started the dht
2019-06-17 14:08:44,505 INFO     torba.client.basedatabase:189: connecting to database: /storage/emulated/0/Android/data/io.lbry.browser/files/lbryum/lbc_mainnet/blockchain.db
2019-06-17 14:08:44,511 INFO     lbrynet.dht.node:118: DHT node listening on UDP 0.0.0.0:4444
2019-06-17 14:08:45,118 INFO     torba.client.basenetwork:108: Successfully connected to SPV wallet server: lbryumx1.lbry.com:50001
2019-06-17 14:08:45,122 INFO     torba.client.baseledger:247: Subscribing and updating accounts.
2019-06-17 14:08:45,264 INFO     torba.client.baseledger:133: lbc_mainnet: added BlockHeightEvent(height=585959, change=3) header blocks, final height 585959
2019-06-17 14:08:46,179 INFO     lbrynet.wallet.ledger:82: Loaded account bYaRm7QufKzqmeBfLctPNArMFnRivhLzdQ with 0.0 LBC, 20 receiving addresses (gap: 20), 6 change addresses (gap: 6), 0 channels, 0 certificates and 0 claims. 
2019-06-17 14:08:46,182 INFO     lbrynet.extras.daemon.Components:432: Starting the file manager
2019-06-17 14:08:46,186 INFO     lbrynet.stream.stream_manager:147: Initializing 0 files
2019-06-17 14:08:46,194 INFO     lbrynet.stream.stream_manager:162: Started stream manager with 0 files
2019-06-17 14:08:46,194 INFO     lbrynet.extras.daemon.Components:438: Done setting up file manager
2019-06-17 14:08:55,212 INFO     torba.client.baseledger:133: lbc_mainnet: added BlockHeightEvent(height=585960, change=1) header blocks, final height 585960
2019-06-17 14:09:11,638 INFO     torba.client.baseledger:133: lbc_mainnet: added BlockHeightEvent(height=585961, change=1) header blocks, final height 585961
2019-06-17 14:10:06,220 INFO     lbrynet.extras.daemon.Daemon:815: Get version info: {"processor": "", "python_version": "3.7.1", "platform": "Android 9 (API 28)", "os_release": "4.4.153-15480849", "os_system": "android", "lbrynet_version": "0.37.4", "torba_version": "0.5.4a0", "build": "release"}
2019-06-17 14:10:13,196 INFO     torba.client.baseledger:133: lbc_mainnet: added BlockHeightEvent(height=585962, change=1) header blocks, final height 585962
2019-06-17 14:11:10,106 INFO     lbrynet.stream.stream_manager:388: starting download for lbry://Why-I-Quit-YouTube#dd87adb49b33f80477050c56bf937880f8f4d18c
2019-06-17 14:11:10,108 INFO     lbrynet.stream.managed_stream:274: start downloader for stream (sd hash: c001e9bfc74568d85fa4e93a36472639f4e76682302c664fe3444ba09bfa0c6c18951696b076293f4d20d61cff7349aa)
2019-06-17 14:11:10,215 INFO     lbrynet.stream.downloader:102: searching for peers for stream c001e9bfc74568d85fa4e93a36472639f4e76682302c664fe3444ba09bfa0c6c18951696b076293f4d20d61cff7349aa
2019-06-17 14:11:40,125 WARNING  lbrynet.extras.daemon.Daemon:937: Error downloading lbry://Why-I-Quit-YouTube#dd87adb49b33f80477050c56bf937880f8f4d18c: Failed to download sd blob c001e9bfc74568d85fa4e93a36472639f4e76682302c664fe3444ba09bfa0c6c18951696b076293f4d20d61cff7349aa within timeout
```

**Comment by tzarebczan** [comment]
Even tried on port 443 on DHT, and it still would not connect. It must be related to UDP traffic somehow.

**Comment by tzarebczan** [comment]
Sounds like @jackrobison / @kauffj got to the bottom of this and it's related to IPV6. Let's get this into a mobile release ASAP.

**Comment by kauffj** [comment]
Btw if this improves reliability measurably I am now the lead data network developer.

**Comment by tzarebczan** [comment]
Can't wait to see the results! Jeremy confirmed this will be fixed in the next mobile release with 0.43.1
